run_itest.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if [ -z ${TRAVIS_PULL_REQUEST} ] || [ ${TRAVIS_PULL_REQUEST} = "false" ]; 
+then
+  echo "Missing env TRAVIS_PULL_REQUEST.  Must be run from a PR."
```

Comment:
Wouldn't be worth to run these itest also in master branch? or these tests are going to be exclusive for posting the screenshot?

Reply:
we can run on PR build for now.  I just pushed a new small update to the if statement